packages/libs/codemodel/src/model/common/parameter.ts

```diff
@@ -29,6 +29,11 @@ export interface Parameter extends Value {
 
   /** When a parameter is grouped into another, this will tell where the parameter got grouped into. */
   groupedBy?: Parameter;
+
+  /**
+   * If this parameter is to be included in a multipart request body.
+   */
+  isInMultipart?: boolean;
```

Comment:
When we update any of the types in `codemodel` we also have to run the `packages/libs/codemodel/.scripts/generate-schema.js` script (no args, just run it with `node`) so that the schema files in `.resources` get updated.